dhall-try/index.html

```diff
@@ -222,18 +225,36 @@ <h1 class="text-center">The Dhall configuration language</h1>
       </div>
     </div>
     <div class="container jumbotron" style="background-color: #c5eccf">
-      <h1 class="display-4">Human-friendly</h1>
-      <p class="lead">Dhall supports comments, multi-line string literals and
-      string interpolation with non-technical users in mind.</p>
-      <textarea id="highlight0"></textarea>
-      <p>You can also automatically remove all indirection in any Dhall code,
-      converting the file to a logic-free normal form for non-programmers to
-      understand.</p>
-      <a href="https://github.com/dhall-lang/dhall-lang/wiki/Programmable-configuration-files"class="btn btn-lg btn-outline-dark bg-light">Learn More <i class="fas fa-book"></i></a>
+      <h1 class="display-4">Don't repeat yourself</h1>
+      <hr class="my-4">
+      <p class="lead">Struggling with configuration drift?  Create a <a href="https://en.wikipedia.org/wiki/Single_source_of_truth">single source of truth</a> you can reference everywhere.</p>
+      <textarea rows="30" id="highlight0"></textarea>
+      <p>Read the same Dhall configuration file directly into several languages or convert Dhall to JSON/YAML</p>
+      <a href="https://github.com/dhall-lang/dhall-lang/wiki/How-to-integrate-Dhall" class="btn btn-lg btn-outline-dark bg-light">Supported integrations <i class="fas fa-cogs" ></i></a>
+    </div>
+    <div class="container jumbotron" style="background-color: #cfebfb">
+      <h1 class="display-4">Fearlessly refactor</h1>
+      <p class="lead">Need to clean up a big mess?  Move fast without breaking things by leaning on Dhall's tooling.</p>
+      <hr class="my-4">
+      <p>Refactoring something mission-critical?  Use Dhall's support for
+      semantic hashes to guarantee that a refactor is behavior-preserving</p>
+      <img src="./img/hash.gif">
+      <hr class="my-4">
+      <p>What if you intend to make a change?  Use a semantic diff to verify that
+      you changed what you expected to:</p>
+      <textarea id="diff"></textarea>
+      <hr class="my-4">
+      <p>Did you "inherit" a messy configuration?  Use the type system and
```

Comment:
I think "to inherit" is pretty established in this sense.

```suggestion
      <p>Did you inherit a messy configuration?  Use the type system and
```